src/commands/dev/dev.js

```diff
@@ -178,18 +178,25 @@ const runCommand = (command, env = {}, spinner = null) => {
   return commandProcess
 }
 
+/**
+ * @typedef StartReturnObject
+ * @property {4 | 6 | undefined=} ipVersion The version the open port was found on
+ */
+
 /**
  * Start a static server if the `useStaticServer` is provided or a framework specific server
  * @param {object} config
  * @param {Partial<import('../../utils/types').ServerSettings>} config.settings
- * @returns {Promise<void>}
+ * @returns {Promise<StartReturnObject>}
```

Comment:
The return value is always an empty object, right?

Reply:
Either empty or has the property `ipVersion`